root/components/EventsList.js

```diff
@@ -138,7 +139,7 @@ const EventsList = ({
             <td>
               {event.performers.map(performer => (
                 performer.entity.id === artist.id ? (
-                  commaOnlyList(performer.roles)
+                  commaOnlyList(localizeArtistRoles(performer.roles))
```

Comment:
We do uniq() in ArtistRoles.js but not here - should we?

Reply:
Not sure - I'd think it should be done on the server in both cases.